executor/api/grpc/kfserving/client.go

```diff
@@ -0,0 +1,187 @@
+package kfserving
+
+import (
+	"context"
+	"fmt"
+	"github.com/go-logr/logr"
+	"github.com/pkg/errors"
+	"github.com/seldonio/seldon-core/executor/api/client"
+	grpc2 "github.com/seldonio/seldon-core/executor/api/grpc"
+	"github.com/seldonio/seldon-core/executor/api/grpc/kfserving/inference"
+	"github.com/seldonio/seldon-core/executor/api/payload"
+	v1 "github.com/seldonio/seldon-core/operator/apis/machinelearning.seldon.io/v1"
+	"google.golang.org/grpc"
+	"io"
+	"math"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+)
+
+type KFServingGrpcClient struct {
+	Log            logr.Logger
+	callOptions    []grpc.CallOption
+	conns          map[string]*grpc.ClientConn
+	Predictor      *v1.PredictorSpec
+	DeploymentName string
+	annotations    map[string]string
+}
+
+func (s *KFServingGrpcClient) ModelMetadata(ctx context.Context, modelName string, host string, port int32, msg payload.SeldonPayload, meta map[string][]string) (payload.ModelMetadata, error) {
+	panic("implement me")
```

Comment:
Nit: MAy be better something like "not implemented", espeically if this returns to the user as 500 error

Reply:
Yep